index.js

```diff
@@ -17,8 +17,7 @@ if (createId) {
   })
 } else if (applyIds) {
   console.log(c.yellow('applying', applyIds))
-  var packager = useYarn ? 'yarn' : 'npm'
-  stack.apply(applyIds, end, packager)
+  stack.apply(applyIds, useYarn ? 'yarn' : 'npm', end)
```

Comment:
This commit is not ok since you commit a state which doesn't match the current version of `.apply()`. Please try to be a bit more patient and commit an acceptable state.

Reply:
You don't have to change anything since I will squash your PR anyway, so no worries :) But for future work, please try to think about this.

Reply:
Yes, i was not doing this fix remotely so i had to do two commits, sorry.